src/biogeochem/CNCStateUpdate1Mod.F90

```diff
@@ -476,13 +546,19 @@ subroutine CStateUpdate1( num_soilc, filter_soilc, num_soilp, filter_soilp, &
                ! isolate, caused C12/C13 ratios to occasionally go out of
                ! bounds. Zeroing out these small pools and putting them into the flux to the
                ! atmosphere solved many of the crop isotope problems
-
                cf_veg%xsmrpool_to_atm_patch(p) = cf_veg%xsmrpool_to_atm_patch(p) + cs_veg%xsmrpool_patch(p)/dt
                cs_veg%xsmrpool_patch(p)        = 0._r8
                cf_veg%xsmrpool_to_atm_patch(p) = cf_veg%xsmrpool_to_atm_patch(p) + cs_veg%cpool_patch(p)/dt
                cs_veg%cpool_patch(p)           = 0._r8
-               cf_veg%xsmrpool_to_atm_patch(p) = cf_veg%xsmrpool_to_atm_patch(p) + cs_veg%frootc_patch(p)/dt
-               cs_veg%frootc_patch(p)          = 0._r8
+               if(.not. use_matrixcn)then
+                  cf_veg%xsmrpool_to_atm_patch(p) = cf_veg%xsmrpool_to_atm_patch(p) + cs_veg%frootc_patch(p)/dt
+                  cs_veg%frootc_patch(p)          = 0._r8
+               else
+                 if(cs_veg%frootc_patch(p) .ne. 0)then
```

Comment:
For consistent formatting please use `/=` instead of   `.ne.`

Reply:
done